R/release.R

```diff
@@ -137,12 +137,12 @@ release_checklist <- function(version, on_cran) {
     "Wait for CRAN...",
     "",
     todo("Accepted :tada:"),
+    todo("Add link to blog post in pkgdown news menu", type != "patch"),
```

Comment:
```suggestion
    todo("Add preemptive link to blog post in pkgdown news menu", type != "patch"),
```

Maybe a little keyword to hint at the fact that the link won't work yet?